frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -33,11 +37,16 @@ type ManageInferenceServiceModalProps = {
   }
 >;
 
-const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> = ({
+export const ManageInferenceServiceModalWrapper: React.FC<
+  ManageInferenceServiceModalProps & { children: React.ReactNode }
+> = ({
```

Comment:
Feel free to leave this as-is, but for each of these modals I wonder if you really need the Wrapper components and introduction of `children` - You could instead just add an optional `projectSection` prop that defaults to the default children you pass below and let the DeployRegisteredModelModal pass its own in that overrides it.

Reply:
Good suggestion, will update!